app/services/importer/ashbyhq_fields_formatter.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+module Importer
+  # This service object is responsible for formatting data received from Ashbyhq, another recruiting platform, into a structured format suitable for our application.
+  # It extracts specific sections (core questions and surveys) and transforms them into a standardized format.
```

Comment:
What is the standardised format that it outputs it in? Can you show me an example?

The point we were discussing with Dan yesterday is why do we standardise the format twice, once in the formatter and again in the field builder? 

Wouldn't it make sense to just format it once?

Reply:
one format, the other build.
it is a separation of concern.
if you ask me to transform application_question_set from array to hash ... at the moment i will have to change only one file.
If we go with your proposition i will have to change every ats_formatter.

Reply:
Can you give an example of what the formatter outputs?